infra/bridge/src/utils/log-chat.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const sendgridMail = require('@sendgrid/mail')
+const { TelegramChatLog } = require('../models/index')
+
+const logger = require('../logger')
+
+module.exports = async message => {
```

Comment:
nit: it's fairly obvious but add a comment explaining the purpose of this method